terraform/mesh_aws/lambda_check_send_parameters.tf

```diff
@@ -0,0 +1,155 @@
+resource "aws_lambda_function" "check_send_parameters" {
+  function_name    = "${local.name}-check-send-parameters"
```

Comment:
"function_name" is being defined here but also further down as cloudwatch log group name.
This should be defined as a "local" at the head of this file (and for all other lambdas)
Then this local can be used for both lambda function and cloudwatch log group name
This removes the risk of someone changing just the lambda name without changing the log group name in the future